.github/workflows/ci.yml

```diff
@@ -67,7 +67,7 @@ jobs:
 
   protobuf_backward_compat:
     name: "Protobuf Backward Compatibility"
-    runs-on: ubuntu-latest
+    runs-on: warp-ubuntu-2204-x64-2x
```

Comment:
Why move off free runners for things like these? We've already made sure that we only run quick/cheap work on these.